components/ProposalRow.tsx

```diff
@@ -128,6 +128,7 @@ export default function ProposalRow({
                       className="rounded bg-indigo-600 px-2 py-1 text-xs font-semibold text-white shadow-sm hover:bg-indigo-500 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-indigo-600"
```

Comment:
_(random placement)_

When connected to goerli on the bridge page, clicking `switch direction` causes the error below:
<img width="880" alt="image" src="https://github.com/ScopeLift/l2-flexible-voting-frontend/assets/61768337/22994a69-d1c5-4013-aa36-9d933ef1d2ed">

Reply:
Same error if I switch to Optimism Goerli from the network dropdown or if I was already on Optimism Goerli on Metamask

Reply:
Interesting that this is only happening to one of my tet wallets: 0x637C1Ec1d205a4E7a79c9CE4Bd100CD1d19E6080

Reply:
thanks @garyghayrat , i ironed out this edge case in latest revision!